monitor/coredns/monitor-coredns-daemonset.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: monitor-coredns-daemonset
+  namespace: metrics-debug
+  labels:
+    app: monitor-coredns-daemonset
+spec:
+  selector:
+    matchLabels:
+      app: monitor-coredns-daemonset
+  template:
+    metadata:
+      labels:
+        app: monitor-coredns-daemonset
+    spec:
+      # tolerations:
+      ## this toleration is to have the daemonset runnable on master nodes
+      ## remove it if your masters can't run pods
+      # - key: node-role.kubernetes.io/master
+      #  effect: NoSchedule
+      hostPID: true
+      hostIPC: true
+      hostNetwork: true
+      containers:
+      - name: monitor-coredns-daemonset
+        image: quay.io/support-tools/monitor-coredns:latest
+        command:
```

Comment:
Its fine if we fixed this in subsequent commits but  wouldn't it better to have /entrypoint/collect-mertrics.sh to sleep infinity and wait for sigterm/ int, so that we don't have to override the entrypoint in daemonset?

Reply:
This should be fixed in the image so I have removed the entrypoint definition.